share/functions/fish_vi_cursor.fish

```diff
@@ -92,7 +92,7 @@ function fish_vi_cursor -d 'Set cursor shape for different vi modes'
 
     echo "
           function fish_vi_cursor_handle_preexec --on-event fish_preexec
-              set -l varname fish_cursor_default
+              set -l varname fish_cursor_external
```

Comment:
This breaks every existing setup that expects the default key to carry over.

What it *should* do is to use the first of fish_cursor_external, fish_cursor_default and fish_cursor_unknown that is defined.

It would also need documentation.

Reply:
Thank you for your feedback!